tools/view.js

```diff
@@ -395,8 +395,35 @@ View.prototype.displaySuspendEvent = function (data, getSource) {
     annotateArray(annotationArray, source.id, data.sections);
     sourceFile.html(arrayToString(annotationArray));
 
+    var sendOperator = sourceFile.find(".EventualMessageSend");
```

Comment:
would be nice if this code could be put into its own function. we got here more than 10 lines of code. Think that justifies giving it a proper name and its own function :)